src/stylesheets/base/_typography.scss

```diff
@@ -1,17 +1,14 @@
 .hdr-text {
-  @include text( $primary-font, 400, 87.5%, $color-dark-green );
+  @include text( $primary-font, 500, 1rem, $color-dark-green );
   text-transform: uppercase;
-  letter-spacing: 2px;
+  letter-spacing: .2rem;
```

Comment:
This is a good use of `rem`, I think, so that this spacing scaled with the text :+1:

Reply:
Actually... maybe that should be `em` so it scales? I got mixed up - rems won't scale 😅